search-app/docs/api.md

```diff
@@ -2950,6 +2966,125 @@ Content-Length: 292
 }
 ```
 
+### <a name="variable"></a> Variable
```

Comment:
This section is a little bit tricky. We do not support creating variables through search (only creating variable associations). I think the section on variables should be moved to the ingest documentation and the associations should stay here.